walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -21,75 +21,64 @@
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
+import static com.google.sps.TimeRange.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    List<TimeRange> possibleTimes = new ArrayList<TimeRange>();
-    Set<String> attendees =  new HashSet<String>();
-    Set<String> optionalAttendees =  new HashSet<String>();
-    int meetingDur = (int) request.getDuration();
-    attendees.addAll(request.getAttendees());
-    optionalAttendees.addAll(request.getOptionalAttendees());
-    if (meetingDur > TimeRange.WHOLE_DAY.duration()) {
-      return possibleTimes;
+    int meetingDuration = (int) request.getDuration();
```

Comment:
It normally returns a long, but I think based off of your comment below I could refactor this chunk so that I don't need the extraneous base case checks in this method which would mean that this variable wouldn't be used for the same purpose!